### PR TITLE
Fix dead link in update doc

### PR DESCRIPTION
**Pull request by DaedalusG**
## Test plan

Docs update

### CODE REVIEW

File: doc/admin/deploy/kubernetes/update.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ To perform a multi-version upgrade on a Sourcegraph instance running on Kubernet
...
       1. Leave these versions of the databases running while the subsequent migration steps are performed. If `codeinsights-db` is a container new to your instance, now is a good time to start it as well.
 1. Pull the upstream changes for the target instance version and resolve any git merge conflicts. The [standard upgrade procedure](#standard-upgrades) describes this step in more detail.
-1. Follow the instructions on [how to run the migrator job in Kubernetes](http://localhost:5080/admin/how-to/manual_database_migrations#kubernetes) to perform the upgrade migration. For specific documentation on the `upgrade` command, see the [command documentation](./../../how-to/manual_database_migrations.md#upgrade). The following specific steps are an easy way to run the upgrade command:
+1. Follow the instructions on [how to run the migrator job in Kubernetes](https://sourcegraph.com/admin/how-to/manual_database_migrations#kubernetes) to perform the upgrade migration. For specific documentation on the `upgrade` command, see the [command documentation](./../../how-to/manual_database_migrations.md#upgrade). The following specific steps are an easy way to run the upgrade command:
```
Comment (efritz) [inline doc/admin/deploy/kubernetes/update.md:85]: NO WAY I DID THIS

**Review by efritz — APPROVED** [review]

File: doc/admin/deploy/kubernetes/update.md (hunk 1 above)
Comment (DaedalusG) [reply doc/admin/deploy/kubernetes/update.md:85]: TOO MUCH UPDATES, STOP THE RECURSION

Comment (DaedalusG) [reply doc/admin/deploy/kubernetes/update.md:85]: ERIC DISK FRAGMENT PLZ HELP

Comment (efritz) [reply doc/admin/deploy/kubernetes/update.md:85]: 🤖

Comment (airamare01) [reply doc/admin/deploy/kubernetes/update.md:85]: WAY TO SPOT THIS TOO LIZARD MAN!

**Pull request merged**